.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+`<description>`
+
+## Steps to Test
+
+1. `yarn && yarn setup`
+2. `yarn test packages/<composites|sources|targets>/<adapter-name>/test`
+3. ...
+
+## New Adapter Checklist
+
+- [ ] Adapter was created by following [Creating a New Adapter](../../CONTRIBUTING.md#creating-a-new-adapter)
+- [ ] Endpoints are built following the style of the [example adapters](../../packages/examples).
+- [ ] Unit and integration tests cover all helper methods and endpoints.
+- [ ] Input/outputs are built to spec.
+- [ ] Code is documented with `const description = "..."` in each endpoint, and `description: "..."` in `schemas/env.json`.
+- [ ] Input parameters are defined in `const inputParameters = {...}` and `export type TInputParameters = {...}` in each endpoint file.
+- [ ] All environment variables are defined in `schemas/env.json` (ex. `API_KEY` and `API_SECRET` if applicable).
+- [ ] `config/` contains rate limits in `limits.json`, symbol overrides in `overrides.json`, and special param "includes" in `includes.json` (only a subset of these may be required for a given EA).
+- [ ] Adapter package version set to `0.0.0`.
+- [ ] 2 Changesets: 1 Major upgrade for the new EA itself, and 1 Minor upgrade for `@chainlink/ea` regarding the addition of a new adapter to `legos`.
```

Comment:
Maybe we can add a link to a section explaining how to do this

Reply:
Done!